compose-builder/asc-external-mqtt-trigger.env

```diff
@@ -0,0 +1,25 @@
+# /*******************************************************************************
+#  * Copyright 2021 Intel Corporation.
+#  *
+#  * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+#  * in compliance with the License. You may obtain a copy of the License at
+#  *
+#  * http://www.apache.org/licenses/LICENSE-2.0
+#  *
+#  * Unless required by applicable law or agreed to in writing, software distributed under the License
+#  * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  * or implied. See the License for the specific language governing permissions and limitations under
+#  * the License.
+#  *
+#  *******************************************************************************/
+#
+# This file contains the common environment overrides used by App Service MQTT export.
+#
+
+EDGEX_PROFILE=external-mqtt-trigger
+TRIGGER_EXTERNALMQTT_URL=tcp://MQTT_BROKER_ADDRESS_PLACE_HOLDER:1883
+WRITABLE_PIPELINE_FUNCTIONS_MQTTEXPORT_PARAMETERS_BROKERADDRESS=tcp://MQTT_BROKER_ADDRESS_PLACE_HOLDER:1883
+WRITABLE_PIPELINE_FUNCTIONS_MQTTEXPORT_PARAMETERS_TOPIC=edgex-export
+# allows scripts to find and change with sed
+WRITABLE_LOGLEVEL=INFO
```

Comment:
Looks like the settings are only use in the one yaml file, so we could reduce number of files by moving these into the environment section of the yaml file.

Reply:
Moved the environment variables to the compose-builder/add-asc-external-mqtt-trigger.yml file.